### PR TITLE
Added File Service with methods Crud to interact with database 

### PR DESCRIPTION
**Pull request by Laura4lilavati**

### CODE REVIEW

File: src/main/java/org/fundacionjala/converter/Application.java [hunk 1]
```diff
@@ -18,49 +18,39 @@ public static void main(final String[] args) {
...
-        System.out.println();
-
-        // total Files in DB
-        System.out.println("Total Files in DB:");
-        System.out.println("------------------");
-        System.out.println(fileRepository.count());
+        fileService.updateFile(fileUpdate);
         System.out.println();
         };
     }
```
Comment (julia-jpg) [inline src/main/java/org/fundacionjala/converter/Application.java]: remove the code from this class, please

File: src/main/java/org/fundacionjala/converter/model/service/FileService.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    /**
+     * Returns element asociated to the given id
+     * @param i - the Long id of element to show
+     * @return file - a reference to the file from the table
+     */
+    public File getFileById(final int i) {
+        Optional<File> file = fileRepository.findById((long) i);
+        if (file.isPresent()) {
+            return file.get();
+        } else {
```
Comment (julia-jpg) [inline src/main/java/org/fundacionjala/converter/model/service/FileService.java:38]: the parameter "i" you can change by " index" line34,35

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -12,15 +12,15 @@ inputFiles.path=storage/inputFiles/
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.datasource.username=root
 spring.datasource.password=root
```
Comment (julia-jpg) [inline src/main/resources/application.properties]: you could delete the password for more convenience hehe

File: src/main/java/org/fundacionjala/converter/model/service/FileService.java (hunk 2 above)
Comment (Laura4lilavati) [reply src/main/java/org/fundacionjala/converter/model/service/FileService.java:38]: it's done

File: src/main/resources/application.properties (hunk 3 above)
Comment (Laura4lilavati) [reply src/main/resources/application.properties]: it's done

**Pull request merged**